akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorSpec.scala

```diff
@@ -704,9 +702,35 @@ class EventSourcedBehaviorSpec extends ScalaTestWithActorTestKit(EventSourcedBeh
 
     }
 
+    "fail fast if persistenceId is null" in {
+      intercept[IllegalArgumentException] {
+        PersistenceId(null)
+      }
+      val probe = TestProbe[AnyRef]
+      EventFilter[ActorInitializationException](start = "persistenceId must not be null", occurrences = 1).intercept {
+        val ref = spawn(Behaviors.setup[Command](counter(_, persistenceId = PersistenceId(null))))
+        probe.expectTerminated(ref)
+      }
+      EventFilter[ActorInitializationException](start = "persistenceId must not be null", occurrences = 1).intercept {
+        val ref = spawn(Behaviors.setup[Command](counter(_, persistenceId = null)))
+        probe.expectTerminated(ref)
+      }
+    }
+
+    "fail fast if persistenceId is empty" in {
+      intercept[IllegalArgumentException] {
+        PersistenceId("")
+      }
+      val probe = TestProbe[AnyRef]
+      EventFilter[ActorInitializationException](start = "persistenceId must not be empty", occurrences = 1).intercept {
+        val ref = spawn(Behaviors.setup[Command](counter(_, persistenceId = PersistenceId(""))))
+        probe.expectTerminated(ref)
+      }
+    }
+
```

Comment:
there was already a test corresponding to "support calling persistAll with Nil", here named "chainable side effects without events"